addons/controllers/cpi/cpiconfig_datavalues.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package controllers
+
+// CPIDataValues is the data values type generated by the CPIConfig CR
+type CPIDataValues struct {
+	VSphereCPI CPIDataValuesVsphereCPI `yaml:"vsphereCPI"`
+}
+
+type CPIDataValuesVsphereCPI struct {
+	Mode                               string      `yaml:"mode"`
+	TLSThumbprint                      string      `yaml:"tlsThumbprint"`
+	Server                             string      `yaml:"server"`
+	Datacenter                         string      `yaml:"datacenter"`
+	Username                           string      `yaml:"username"`
+	Password                           string      `yaml:"password"`
+	Region                             interface{} `yaml:"region"`
```

Comment:
Question: What's the reason of using `interface{}` here?

Reply:
to match the package value definition here https://github.com/vmware-tanzu/community-edition/blob/8e903a7fe266432e6c5515441ecf56cfdcb9dc76/addons/packages/vsphere-cpi/1.22.5/bundle/config/values.yaml#L11

Reply:
I think it's fine to use string type even the default values.yaml had it as null. Did you see any error when using string?

Reply:
no, I didn't observe any error for now. Let me change it to string

Reply:
changed to string